src/docs/metrics/eth-2/index.md

```diff
@@ -8,12 +8,22 @@ description: ETH 2.0 Metrics
 ## Description
 Metrics that show stats related to the ETH 2.0 Staking Contract
 
-* `eth2_stakers_count` - amount of addresses that sent ETH to the ETH 2.0 Staking Contract
-* `eth2_roi` - the annual return on staking in the ETH 2.0 contract
-* `eth2_staked_amount_per_label` - total staked amount by label
-* `eth2_staked_address_count_per_label` - amount of addresses that staked ETH by label
-* `eth2_unlabeled_staker_inflow_sources` - shows sources from which unlabeled stakers received ETH
-* `eth2_top_stakers` - top addresses by staked amount
+* `eth2_stakers_count` - Amount of addresses that sent ETH to the ETH 2.0 Staking Contract
+* `eth2_top_stakers` - Top addresses by staked amount
```

Comment:
The histogram metrics must have an example how to use them. They cannot be queried via the standard `getMetric { timeseriesData }` as they have special return format, which users are not able to come up with by themselves.

You can get the examples from here: https://github.com/santiment/sanbase2/pull/2399

Maybe it would be good to come up with a way to specify this. Something like:
```
 `eth2_top_stakers` - Top addresses by staked amount (histogram metric - see examples for details how to query it)
 ```

Reply:
Added `ETH 2.0 histogram metrics` group